apis/types/container_copy.go

```diff
@@ -0,0 +1,30 @@
+package types
```

Comment:
We do not manually edit this file, but generate it by swagger, you could see the reference: https://github.com/alibaba/pouch/blob/master/apis/README.md

Reply:
ok